extension/textext/asktext.py

```diff
@@ -246,6 +169,18 @@ def ask(self, callback, preview_callback=None):
         """
         pass
 
+    def error_dialog(self, title, label, error):
+        """
+        Presents an error dialog
+
+        :param parent: Parent window
+        :param title: Error title text
+        :param label: Label text
+        :param error: exception
+        :type error: StandardError
```

Comment:
Wouldn't the consequent usage of``abstractmethod``be more consistent?

Reply:
I think it's not worth here. The base class and Tk live their last days before we switch to GTK3 completely. 

In principle `abc` provides means to express class hierarchy in more details than bare `raise NotImplementedError`. Here we are authors and the only consumers of the class, no need to overdocument it. 

I don't like `pass` because it says "you are free to implement this method, but you not have to"

Reply:
:heavy_check_mark: